packages/teleterm/src/ui/TopBar/TopBar.tsx

```diff
@@ -33,24 +45,11 @@ const Grid = styled.div`
 `;
 
 const CentralContainer = styled.div`
-  display: grid;
-  column-gap: 10px;
-  margin: auto 10px;
-  grid-template-columns: minmax(150px, 280px) minmax(200px, 600px);
-  align-items: center;
-  height: 100%;
-`;
-
-const JustifyLeft = styled.div`
   display: flex;
-  justify-self: start;
-  align-items: center;
-  height: 100%;
-`;
-
-const JustifyRight = styled.div`
-  display: grid;
-  justify-self: end;
+  flex: 1;
+  margin-left: 22px;
+  margin-right: 10px;
```

Comment:
Could you change these values so that they're multiples of 4? It'll be easier to adjust the layout later if we keep this rule.

```suggestion
  margin-left: 24px;
  margin-right: 12px;
```